src/services/__tests__/messagesService.test.ts

```diff
@@ -68,6 +82,29 @@ const validApiMessageResponse = {
     status: "PROCESSED"
   }
 };
+
+const validApiWithAttachmentsMessageResponse = {
+  status: 200,
+  value: {
+    message: {
+      content: {
+        attachments: [aValidNrePngBarcode, aValidNreSvgBarcode],
```

Comment:
sorry I've review the test and my first suggestion was wrong. the test must be rewritten anyway:
`validApiWithAttachmentsMessageResponse` is the response we get from io-functions-app which does not contains the `attachments` field BUT contains the `prescription_metadata` values. the test must state that when we get `prescription_metadata` in input we get the correct `attachments` output

Reply:
fixed here: https://github.com/pagopa/io-backend/pull/648/commits/c18e7cb45346d7c8b7d11f09a99b389c371bb50b?file-filters%5B%5D=.ts